cluster/src/main/java/io/scalecube/cluster/ClusterImpl.java

```diff
@@ -118,7 +120,6 @@ public ClusterImpl(ClusterConfig config) {
                   Collections.singletonList(
                       membership
                           .listen()
-                          .publishOn(scheduler)
```

Comment:
I removed this line because: 
subscriber would still be executing in cluster algorithm thread (not in netty thread!) because when memebership listen from downstream components such as gossip and fd then we still doing `.publishOn(scheduler).subscribe()`